sklearn/utils/validation.py

```diff
@@ -295,8 +295,10 @@ def check_array(array, accept_sparse=None, dtype="numeric", order=None,
         If dtype is a list of types, conversion on the first type is only
         performed if the dtype of the input is not in the list.
 
-    order : 'F', 'C' or None (default=None)
-        Whether an array will be forced to be fortran or c-style.
+    order : 'F', 'C' or 'A' (default='A')
```

Comment:
Actually, `copy=True` "tries" to preserve order, 

https://github.com/numpy/numpy/issues/7374#issuecomment-191919769